### ISSUE TITLE
Rotate the doubly Linked list with n no nodes provided

### ISSUE THREAD

**Issue by harsh6768-svg** [Ayush7614/Daily-Coding-DS-ALGO-Practice#1233]
# Aim

- Most Famous Question of GFG asked many times in interview also

# Details

- We will be provided a doubly linked ist and with n no of swaps 
- We will try to rotate the linked list
- For example:
- Input : a  b  c  d  e  f  g  h   N = 4
Output : e  f  g  h  a  b  c  d 

### Programming language

- [ ] C
- [ ] C++
- [ ] Java
- [ ] Python

**Comment by shruti29s** [comment]
Can I work on this issue ?

**Comment by harsh6768-svg** [comment]
But I m already working @shruti29s

**Comment by shruti29s** [comment]
I can do this in java.

**Comment by harsh6768-svg** [comment]
okkk please confirm with @Ayush7614 bhaiya

**Comment by shruti29s** [comment]
Yes, sure. I will work on this only when someone assign me this issue.
